.ruff.toml

```diff
@@ -0,0 +1,10 @@
+target-version = "py311"
```

Comment:
```suggestion
target-version = "py310"
```

Reply:
Since we have adopted `py311` in EDM4hep, I think we can also leave it here.

Reply:
Yes